v3/how-to-guides/01-basics/e-custom-chainspec/index.mdx

```diff
@@ -48,26 +56,26 @@ We have just generated a **plain chain spec** file for the _default_ network set
 
 ### 2. Modify the plain chain specification (optional)
 
-This optional step we can leverage an _existing_ plain chain specification for a network that otherwise would require modifiaciton of te _source_ of the node to run on a _new network_.
-For example, this can be quite usefull in the [Cumulus Tutorial](/tutorials/v3/cumulus/start-relay) where we want to create a custom _relay chain_ without customizing Polkadot's source.
+This optional step we can leverage an _existing_ plain chain specification for a network that otherwise would require modification of te _source_ of the node to run on a _new network_.
```

Comment:
```suggestion
This optional step we can leverage an _existing_ plain chain specification for a network that otherwise would require modification of the _source_ of the node to run on a _new network_.
```